src/app.js

```diff
@@ -1 +1,33 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+function infinityCreation() {
+  const myDate = new Date();
+  const currentTime = `${
+    ('0' + myDate.getHours()).slice(-2)}_${
+    ('0' + myDate.getMinutes()).slice(-2)}_${
+    ('0' + myDate.getSeconds()).slice(-2)}`;
+
+  const currentTimeForConsole = `${
+    ('0' + myDate.getHours()).slice(-2)}:${
+    ('0' + myDate.getMinutes()).slice(-2)}:${
+    ('0' + myDate.getSeconds()).slice(-2)}`;
```

Comment:
```suggestion
  const currentTimeForConsole = `${
    ('0' + myDate.getHours()).slice(-2)}:${
    ('0' + myDate.getMinutes()).slice(-2)}:${
    ('0' + myDate.getSeconds()).slice(-2)}`;
```

use date.getTime() to get a timestamp of current date